### PR TITLE
added new resource

### PR DESCRIPTION
**Pull request by ujjawaltyagii**
### Closes : #378 

- Added new resource in category ``Design Inspiration`` in Frontend.

## Screenshots
image

### CODE REVIEW

File: database/frontend/design-inspirations.json [hunk 1]
```diff
@@ -123,5 +123,12 @@
         "url": "https://www.saasui.design/",
         "category": "frontend",
         "subcategory": "design-inspiration"
+    },
+    {
+        "name": "Screenlane",
+        "description": "Find app screens, landing pages, user flow inspirations and keep up with the latest mobile UI design trends.",
```
Comment (CBID2) [inline database/frontend/design-inspirations.json:129]:
```suggestion
        "description": "You can find app screens, landing pages, user flow inspirations, and keep up with the latest mobile UI design trends.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @ujjawaltyagii! :) I made some suggestions on how to improve the description. You're doing a great job! 😄 Keep it up! 👍🏾

**Review by CBID2 — APPROVED** [review]
Well done @ujjawaltyagii! LGTM! 😄  👍🏾

**Review by CBID2 — APPROVED** [review]
Looks great @ujjawaltyagii! LGTM! 😄

**Comment by ujjawaltyagii** [comment]
@rupali-codes

**Review by rupali-codes — APPROVED** [review]
Hey Thanks, lgtm

**Pull request merged**